bin/run.js

```diff
@@ -507,6 +513,11 @@ const runCoverageReportOnly = (options, code, signal) => {
 
   const args = [nycBin, 'report', '--reporter', options.coverageReport]
 
+  if (options.typescript) {
+    args.push('--reporter');
+    args.push('text');
+  }
```

Comment:
I don't understand this.  Why are you forcing coverage with a text reporter when `--type-script` is enabled?

Reply:
@isaacs for some reason, when --ts is on, nyc doesn't show coverage in terminal but builds coverage into specified folder. It's just discrepancy that I've found